tests/unit/style/properties/scaffolding.rs

```diff
@@ -25,10 +25,11 @@ fn properties_list_json() {
         .status()
         .unwrap();
     assert!(status.success());
-    let properties = Json::from_reader(&mut File::open(json).unwrap()).unwrap();
+
+    let properties: Value = serde_json::from_reader(File::open(json).unwrap()).unwrap();
```

Comment:
You could deserialize directly to [`serde_json::Map<String, Value>`](https://docs.serde.rs/serde_json/struct.Map.html) instead of `Value` and save those `as_object().unwrap()` calls.